### DISCUSSION TITLE
Position is off if the content of the tootip's height changes

### DISCUSSION THREAD

**Discussion by akraines** [ReactTooltip/react-tooltip#991] [Q&A]
It seems that position is off if the content of the tooltip's height changes. Is there a way to indicate to the tooltip to recalculate its position based on the height of the dynamic content of the tooltip?
I'm trying to work out why tooltips are being positioned way out of place on first render of the cell in my app. On the second render it positions itself correctly. I have considered 2 possibilities:
1. The tooltip is associated with a row that is being rendered as part of a virtual list.
2. The height of the tooltip changes as part of its first render [it first says loading and then displays the content.] As a test I gave the tooltip a fixed height and it seemed to fix the issue.

**Comment by gabrieljablonski** [comment]
By default, the tooltip should position itself correctly when using dynamic content. According to your description, this seems like a bug.

Please create a minimal example using [StackBlitz](https://stackblitz.com/fork/react-ts) or something similar and create a new issue demonstrating the problem.

**Comment by akraines** [comment]
Here is a repro: https://stackblitz.com/edit/react-ts-ay7kkp?file=App.tsx
